### PR TITLE
Adding sample ADL settings to documentation

### PR DESCRIPTION
**Pull request by jenniferdai**
Updating documents

### CODE REVIEW

File: docs/pluggable_storage.rst [hunk 1]
```diff
@@ -57,6 +57,9 @@ AzurePinotFS requires the following configs according to your environment:
 
 adl.accountId, adl.authEndpoint, adl.clientId, adl.clientSecret
 
+You can find the parameters in your account as follows:
+https://stackoverflow.com/questions/56349040/what-is-clientid-authtokenendpoint-clientkey-for-accessing-azure-data-lake
```
Comment (mayankshriv) [inline docs/pluggable_storage.rst]: This does not help in showing an example of how the Pinot ADLS config looks like, right?

File: docs/pluggable_storage.rst [hunk 2]
```diff
@@ -57,6 +57,25 @@ AzurePinotFS requires the following configs according to your environment:
 
 adl.accountId, adl.authEndpoint, adl.clientId, adl.clientSecret
 
+Sample Controller Config
```
Comment (mayankshriv) [inline docs/pluggable_storage.rst]: Can we format these configs as JSON?

Comment (jenniferdai) [reply docs/pluggable_storage.rst]: To format as JSON, it requires {}, and I don't want to confuse our users, as just the flat config without the braces is what they need to put in their configuration file

Comment (mayankshriv) [reply docs/pluggable_storage.rst]: Oh I see, this is not JSON config, but apache PropertiesConfiguration file. In that case, this is fine. Could you please add instruction on how this file is passed to `pinot-server`? Also, please make sure that the output html page is formatted correctly.

Comment (jenniferdai) [reply docs/pluggable_storage.rst]: I verified that the output is formatted. Can you clarify what you mean by instruction on how the file is passed? It's the config file that users can save and input to startServer, so we don't do anything special (they essentially just need to declare this path for their config)

File: docs/pluggable_storage.rst (hunk 1 above)
Comment (jenniferdai) [reply docs/pluggable_storage.rst]: It displays where clients need to look for params, which could be helpful for clients confused about meaning

**Pull request merged**